src/ignitus-Shared/Components/ignitus-DashBoard-Footer/DashboardFooter.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import './style.scss';
+
+export default function Footer() {
+  return (
+    <div className="rectangle">
+      <div className='top-row'>
+        <div className='link' id='about'>
+          <a href=''>
+            About
+          </a>
+        </div>
+
+        <div className='link' id='privacy'>
+          <a href=''>
+            Privacy
+          </a>
+        </div>
+
+        <div className='link' id='careers'>
+          <a href=''>
+            Careers
+          </a>
+        </div>
+      </div>
+
+      <div className='bottom-row'>
+        <div className='link' id='helpcenter'>
+          <a href=''>
+            Help Center
+        </a>
+        </div>
+
+        <div className='link' id='terms'>
+          <a href=''>
+            Terms & Conditions
+        </a>
+        </div>
+      </div>
+
+      <div className='link' id='ignitus'>
+        © Ignitus 2018
+      </div>
```

Comment:
@divyanshu-rawat  Can we use `{new Date().getFullYear()}`  so that we don't have to change it manually every time.

Reply:
But I think it reflects when we started building ignitus and that is 2018.